woocommerce/woocommerce
# only activate WooCommerce in E2E initialization

### All Submissions:


### Changes proposed in this Pull Request:

The E2E initialization uses `wp plugin install woocommerce --activate` to activate WooCommerce for the test run. This logs a warning that the plugin is already installed because the current branch is mapped into the plugins folder.

```
Warning: woocommerce: Plugin already installed.
Activating 'woocommerce'...
```

This PR updates the command to just activate WooCommerce

### How to test the changes in this Pull Request:

1. In master run `npm run docker:up`
2. Run `docker ps`
3. Run `docker logs {container id}` where container id is the `cli` container
4. Repeat in this branch
5. You should not see the `already installed` warning
6. Run E2E tests

### Other information:




### Changelog entry

N/A
